cli/cmd/root.go

```diff
@@ -13,25 +13,28 @@ var cfgFile string
 var controlPlaneNamespace string
 var apiAddr string // An empty value means "use the Kubernetes configuration"
 var kubeconfigPath string
-var logLevel string
+var verbose bool
 
 var RootCmd = &cobra.Command{
 	Use:   "conduit",
 	Short: "conduit manages the Conduit service mesh",
 	Long:  `conduit manages the Conduit service mesh.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		// set global log level
-		level, err := log.ParseLevel(logLevel)
-		if err != nil {
-			log.Fatalf("invalid log-level: %s", logLevel)
+		// clear log color formatting
+		log.SetFormatter(&log.TextFormatter{DisableColors: true})
```

Comment:
🚫 🌈 😢

Reply:
Ah, you're right -- it turns out we don't actually need to disable colors. Will remove this.